doc/src/manual/getting-started.md

```diff
@@ -14,9 +14,9 @@ $ julia
   (_)     | (_) (_)    |  Documentation: https://docs.julialang.org
    _ _   _| |_  __ _   |  Type "?help" for help.
   | | | | | | |/ _` |  |
-  | | |_| | | | (_| |  |  Version 0.5.0-dev+2440 (2016-02-01 02:22 UTC)
- _/ |\__'_|_|_|\__'_|  |  Commit 2bb94d6 (11 days old master)
-|__/                   |  x86_64-apple-darwin13.1.0
+  | | |_| | | | (_| |  |  Version 0.7.0-DEV.4480 (2018-03-07 11:38 UTC)
+ _/ |\__'_|_|_|\__'_|  |  Commit fa462ba112 (5 days old master)
```

Comment:
You can replace this full code block with:
````
```@eval
io = IOBuffer()
Base.banner(io)
banner = String(take!(io))
import Markdown
Markdown.parse("""
```
\$ julia

$(banner)
julia> 1 + 2
3

julia> ans
3
```
""")
```
````
and this will be generated when the docs are built and we never have to update this manually again (:tada: )

Reply:
Great idea!